### PR TITLE
PUBDEV-5545: Added section for enforcing system-level commas-line arguments in h2odriver

### PR DESCRIPTION
**Pull request by angela0xdata**

### CODE REVIEW

File: h2o-docs/src/product/security.rst [hunk 1]
```diff
@@ -144,6 +144,27 @@ What is being Secured Today
...
+2. Specify the default command-line options that you want to enforce. Note that each argument must be on a separate line. For example:
+
+ ::
+
+   h2o_ssl_jks_internal=keystore.jks
+   h2o_ssl_jks_password=password
+   h2o_ssl_jts_internal=truststore.jks
+   h2o_ssl_jts_password=password
+
+3. Use the ``-Dai.h2o.args.config`` option to specify the configuration file when starting H2O.
```
Comment (jakubhava) [inline h2o-docs/src/product/security.rst]: This configuration is just to change the default name of the file. The user does not need to specify it if the file `**/etc/h2o/h2odriver.args**` exists

Comment (angela0xdata) [reply h2o-docs/src/product/security.rst]: So can I just remove step 3?

Comment (jakubhava) [reply h2o-docs/src/product/security.rst]: I would keep it, but just write there that `h2odriver.args` is default and people don't have to specify this if they named the file as this, just in cases they used custom name

Comment (michalkurka) [reply h2o-docs/src/product/security.rst]: I would remove it, this is an internal parameter.

On the other hand we should include the description of this functionality in SW as well (with a proper name of the file).

**Pull request merged**